components/cover/basic/src/index.scss

```diff
@@ -0,0 +1,70 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+
+$mah-cover-basic: 676px !default;
+$fz-cover-basic-button: 16px !default;
+
+.sui-CoverBasic {
+  max-height: $mah-cover-basic;
+  position: relative;
+  width: 100%;
+
+  &.is-clickable {
+    cursor: hand;
+  }
+
+  &-gradient {
+    display: inline-block;
+    position: relative;
+    width: 100%;
+  }
+
+  &-gradient:after {
+    background: linear-gradient(to bottom, rgba(0, 0, 0, 0) 25%, rgba(0, 0, 0, 0.75));
+    content: '';
+    display: inline-block;
+    height: 100%;
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 100%;
+  }
+
+  &-image {
+    display: block;
+    height: 100%;
+    object-fit: cover;
+    width: 100%;
+  }
+
+  &-buttonList {
+    display: flex;
+    flex-direction: row;
+    margin: 30px 0 0 52px;
```

Comment:
I think that should come from the designer. They should deliver mockups with normalized spacing... 
Developer should worry about pixel perfect. But, dev should also check if there is a way to do it with our current variables. At least, hardcoded paddings and margins should not happen.
Can't you normalize them with the designer?